### DISCUSSION TITLE
How to downgrade my xformers to 0.0.20

### DISCUSSION THREAD

**Discussion by SekaizaEarthza** [Comfy-Org/ComfyUI#1434] [Q&A] [answered]
I accidentally update it with "update_comfyui_and_python_dependencies". Even I delete Comfy's folder and reinstall it, it doesn't work
![Screenshot 2023-09-06 203851](https://github.com/comfyanonymous/ComfyUI/assets/126684089/c79089d6-5997-4c97-81e0-ec8a82bb0be1)

**Comment by ltdrdata** [comment, marked as answer]
execute this
`.\python_embeded\python.exe -m pip install xformers==0.0.20`

**Reply by SekaizaEarthza** [reply]
![xformers 0 0 20 installation](https://github.com/comfyanonymous/ComfyUI/assets/126684089/83ac69ad-c850-48c2-a660-943fd1c752f7)
![Screenshot 2023-09-07 091403](https://github.com/comfyanonymous/ComfyUI/assets/126684089/97a6dcec-b783-451b-b5ad-a0453fbf67ef)

All done, anything else that I must do? or is there anything goes off?

**Reply by ltdrdata** [reply]
That's fine.

**Reply by SekaizaEarthza** [reply]
Thank you so much. ^^
